acme/commons.go

```diff
@@ -111,7 +111,7 @@ type ExtendedOrder struct {
 }
 
 // Order the ACME order Object.
-// - https://tools.ietf.org/html/draft-ietf-acme-acme-16#section-7.1.3
```

Comment:
Was removing the leading space deliberate?

Reply:
no, it was a mistake